pom.xml

```diff
@@ -32,6 +32,28 @@
 					<artifactId>junit-vintage-engine</artifactId>
 				</exclusion>
 			</exclusions>
+		</dependency>	
```

Comment:
Do not modify previous version of develop branch code. It makes hard to see what your real changes are.